exercises/concept/fibonacci-iterator/.meta/config.json

```diff
@@ -12,7 +12,11 @@
       "fibonacci.jl"
     ]
   },
-  "contributors": [],
+  "contributors": [
+    "cmcaine",
+    "miguelraz",
```

Comment:
This contribution is non-obvious: He solved this on stream which resulted in various improvements to the exercise.